app/index.html

```diff
@@ -33,7 +33,6 @@
   <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.css" />
   <link rel="stylesheet" href="bower_components/metisMenu/dist/metisMenu.css" />
   <link rel="stylesheet" href="bower_components/morrisjs/morris.css" />
-  <link rel="stylesheet" href="bower_components/datatables-responsive/css/dataTables.responsive.css" />
```

Comment:
I rebased with master after which the line was deleted.



On Mon, Aug 1, 2016 at 5:27 PM, nkimaina <notifications@github.com> wrote:

> In app/index.html
> <https://github.com/AMPATH/ng-amrs/pull/519#discussion_r72986631>:
>
> > @@ -33,7 +33,6 @@
> >    <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.css" />
> >    <link rel="stylesheet" href="bower_components/metisMenu/dist/metisMenu.css" />
> >    <link rel="stylesheet" href="bower_components/morrisjs/morris.css" />
> > -  <link rel="stylesheet" href="bower_components/datatables-responsive/css/dataTables.responsive.css" />
>
> Not sure why you need to delete this line
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AMPATH/ng-amrs/pull/519/files/aca3268be63af85fcadfb42727a69bb3f00a54c1#r72986631>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEW5bdo-oifs6EIUhhNGl9oMAjTEvEf9ks5qbgJWgaJpZM4JZSzO>
> .
>